tests/subsys/settings/fcb_init/prj.conf

```diff
@@ -0,0 +1,14 @@
+CONFIG_ZTEST=y
+CONFIG_STDOUT_CONSOLE=y
+CONFIG_FLASH=y
+CONFIG_SOC_FLASH_NRF=y
```

Comment:
Is this SOC_FLASH_NRF=y really necessary? It should be defaulting to 'y' on all nRF targets.